src/PersistenceTests/PersistenceTestsConfiguration.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace NServiceBus.PersistenceTesting
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.ServiceFabric.Data;
+    using NServiceBus.Outbox;
+    using NServiceBus.Sagas;
+    using NUnit.Framework;
+    using Persistence;
+    using Persistence.ServiceFabric;
+    using StatefulService = Microsoft.ServiceFabric.Services.Runtime.StatefulService;
+
+    public partial class PersistenceTestsConfiguration
+    {
+        public bool SupportsDtc { get; } = false;
+
+        public bool SupportsOutbox { get; } = true;
+
+        public bool SupportsFinders { get; } = false;
+
+        public bool SupportsSubscriptions { get; } = false;
+
+        public bool SupportsTimeouts { get; } = false;
+
+        public bool SupportsPessimisticConcurrency { get; } = true;
+
+        public ISagaIdGenerator SagaIdGenerator { get; set; }
+
+        public ISagaPersister SagaStorage { get; set; }
+
+        public ISynchronizedStorage SynchronizedStorage { get; set; }
+
+        public ISynchronizedStorageAdapter SynchronizedStorageAdapter { get; set; }
+
+        public IOutboxStorage OutboxStorage { get; set; }
+
+        public PersistenceTestsConfiguration(TimeSpan timeout)
```

Comment:
this shouldn't be provided. There is already a timeout overload in the partial provided by the package